int32 mode         # 0 = human move, 1 = robot move
int32 symbol       # 0 = O, 1 = X
int32 cell_number  # human move; ignored if robot move

---

bool success
string message
int32 cell_number  # chosen cell number for best move
int32[9] board     # updated board after processing
bool game_over     # gave over, winner detected
int32[3] winner    # {symbol, cell 1, cell 2}